ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/mapper/AmountRangeMapper.java

```diff
@@ -9,7 +9,7 @@
 
     @Override
     public CCDAmountRange to(AmountRange amountRange) {
-        final CCDAmountRange.CCDAmountRangeBuilder builder = CCDAmountRange.builder();
```

Comment:
This is not an argument - are we removing finals from local variables too?

Reply:
Yes

Reply:
@rlewan
Care to elaborate?
When was this decision made and why? Not saying I would oppose, but the name and description of this PR are misleading then.

Reply:
@radoslawn-kainos you are right, I decided to remove final modifier from local variables as well. I have just updated the PR description. Sorry for confusion. The only allowed places for final modifier are:
- class fields (to make classes immutable)
- class definitions (to prevent subclassing)

Reply:
Hi @radoslawn-kainos I agree the name of the PR is not accurate.

We're removing all `final` modifiers from non field references as they give no value and just mean more keystrokes. Awkward code the does something weird with variables reassignment will most likely be caught during code review.

Reply:
Wait. Are we going straight from mandating it for arguments to banning it for all local variables? Isn't that a bit extreme? What if there was a good reason to use it once in a while? If you didn't want/see the need to use one, you wouldn't have to, right? Why make it a policy?